requirements.txt

```diff
@@ -2,5 +2,5 @@ pyee
 requests
 qrcode
 lxml
-wechaty-puppet>=0.0.17
-wechaty-puppet-hostie>=0.2.14
+wechaty-puppet>=0.0.0,<0.1
```

Comment:
0.0.x in semver has special meaning.

I'd like to suggest that we bump the version to 0.1.x

Reply:
Ok, we should bump the version of `python-wechaty-puppet` first.  Please help to merge it.  [pr #32](https://github.com/wechaty/python-wechaty-puppet/pull/32)